NEW_MODULE_TECH_EVAL.MD

```diff
@@ -97,3 +114,7 @@ Information gathered includes:
 
     * Communicates relevent module information through updates to the TCR Issue in jira, once it has been created by the submitter.
     * Includes developers, product owners and other team members.
+
+1. Product Council
+
+    * Leaves a comment in the TCR saying they approved the relevant functionality.
```

Comment:
I am checking with PC on whether this addition is needed, given the new criteria item.  One issue is how PC would indicate that they don't care about a particular module, vs. specifically approving the functionality.

Reply:
Kristin Martin said they might just list "don't care" decisions on the same decision page, so this is moot.